style.css

```
@@ -309,3 +319,58 @@ i:hover {
   margin: 0 1px;
   padding: 0%;
 }
+
+textarea:focus {
+  outline: none;
+  box-shadow: 0 0 5px #60c095;
+}
+
+.container-form input[type="text"] {
+  border: 1px solid #d0d9d4;
+  background-color: #fff;
+  padding: 10px;
+  border-radius: 5px;
+  font-family: 'Inter', sans-serif;
+  width: 300px;
+}
+
+.container-form input[type="email"] {
+  border: 1px solid #d0d9d4;
+  background-color: #fff;
+  padding: 10px;
+  border-radius: 5px;
+  font-family: 'Inter', sans-serif;
+  width: 300px;
+}
```

- [ ] In accordance to CSS best practices, I observed that you're repeating code here. Kindly make the necessary adjustment. ℹ️ 
```suggestion
.container-form input[type="text"],
.container-form input[type="email"] {
  border: 1px solid #d0d9d4;
  background-color: #fff;
  padding: 10px;
  border-radius: 5px;
  font-family: 'Inter', sans-serif;
  width: 300px;
}
```